docs/cmd/kn_source_cronjob_create.md

```diff
@@ -15,7 +15,7 @@ kn source cronjob create NAME --schedule SCHEDULE --sink SINK --data DATA [flags
 ```
 
   # Create a crontab scheduler 'my-cron-trigger' which fires every minute and sends 'ping' to service 'mysvc' as a cloudevent
-  kn source cronjob create my-cron-trigger --schedule "* * * * */1" --data "ping" --sink svc:mysvc
+  kn source cronjob create my-cron-trigger --schedule "*/1 * * * *" --data "ping" --sink svc:mysvc
```

Comment:
```suggestion
  kn source cronjob create my-cron-trigger --schedule "* * * * *" --data "ping" --sink svc:mysvc
```

You can totally omit the "/1" as this is the default anyway.

Reply:
done